src/ai/susi/server/api/cms/ListSkillService.java

```diff
@@ -50,10 +51,8 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
         json.put("accepted", false);
         JSONObject skillObject = new JSONObject();
         ArrayList<String> fileList = new ArrayList<String>();
-        fileList = listFilesForFolder(language, fileList);
```

Comment:
Since fileList is being modified inside the method, hence it doesn't need to be assigned values again.